src/Implement/HttpListener.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "HttpListener.h"
+
+HttpListener *HttpListener::only_http_listener = nullptr;
+
+void HttpListener::InitInstance(
+    std::function<void(const int &, const std::string &)> callback,
+    const int &port_num) {
+  this->callback = callback;
+  this->BindPort(port_num);
+}
+
+// Singleton code convention, must bind with mainManager obj and require
```

Comment:
No need to duplicate function comment in .cpp.

Reply:
Done